source/renderer/app/config/walletsConfig.js

```diff
@@ -32,10 +32,12 @@ export const RECOVERY_PHRASE_WORD_COUNT_OPTIONS = {
 };
 
 export const WALLET_PUBLIC_KEY_NOTIFICATION_SEGMENT_LENGTH = 15;
-export const WALLET_PUBLIC_KEY_SHARING_ENABLED = false;
+export const IS_WALLET_PUBLIC_KEY_SHARING_ENABLED = false;
 
 // Automatic wallet migration from pre Daedalus 1.0.0 versions has been disabled
 export const IS_AUTOMATIC_WALLET_MIGRATION_ENABLED = false;
 
 // Byron wallet migration has been temporarily disabled due to missing Api support after Mary HF
 export const IS_BYRON_WALLET_MIGRATION_ENABLED = false;
+
+export const IS_WALLET_UNDELEGATION_ENABLED = true;
```

Comment:
@yakovkaravelov must not forget to change this to `false` before merging the PR.